public/app/features/dashboard-scene/scene/layout-tabs/TabItem.tsx

```diff
@@ -90,6 +97,26 @@ export class TabItem
     layout.removeTab(this);
   }
 
+  public serialize(): TabsLayoutTabKind {
+    return serializeTab(this);
+  }
+
+  public onCopy() {
+    const panels = this.state.layout.getVizPanels();
+    const scene = this.getRoot();
+    if (!(scene instanceof DashboardScene)) {
+      throw new Error('Scene is not a DashboardScene');
+    }
+    const dsReferencesMapping = scene.serializer.getDSReferencesMapping();
+    const panelsArray = panels.map((vizPanel) => {
+      return vizPanelToSchemaV2(vizPanel, dsReferencesMapping);
+    });
+    const elements = createElements(panelsArray, scene);
+
+    clearClipboard();
+    store.set(LS_TAB_COPY_KEY, JSON.stringify({ elements, tab: this.serialize() }));
+  }
```

Comment:
This code looks close to identical to what's in RowItem, feels like this should be part of the serializer